### ISSUE TITLE
Exclude shadow from semantics. 

### ISSUE THREAD

**Issue by Munsif-Ali** [Munsif-Ali/shadex#1]
The shadow should be excluded from the semantics tree to prevent it from being announced twice by screen readers.
